node/src/reactor/joiner.rs

```diff
@@ -512,8 +507,11 @@ impl reactor::Reactor for Reactor {
                 .expect("should have genesis post state hash"),
             registry,
             Box::new(HighwayProtocol::new_boxed),
-            rng,
         )?;
+        effects.extend(reactor::wrap_effects(
+            Event::Consensus,
+            init_consensus_effects,
+        ));
```

Comment:
What kind of effects do we expect here that we need to handle? Those can only be timers:
1. purge synchronizer timer
2. active validator timer

1) Is not needed since we won't be syncing any protocol state,
2) we won't even activate a validator so those are not needed either.


I think we should be able to delete it entirely.

Reply:
With the changes in this PR, the era supervisor emits a set of storage requests - these are the effects that have to be handled. Without handling these, it can't initialize eras, and syncing breaks.

Reply:
Ah, I forgot about the new effects. Thanks.